search-commons/src/main/java/no/unit/nva/search2/common/AggregationFormat.java

```diff
@@ -137,7 +138,8 @@ static final class Constants {
             CONTRIBUTOR, "/filter/entityDescription/contributor/id",
             CONTEXT_TYPE, "/filter/entityDescription/reference/publicationContext/contextType",
             FUNDING_SOURCE, "/filter/fundings/id",
-            TOP_LEVEL_ORGANIZATION, "/filter/topLevelOrganizations/id"
+            TOP_LEVEL_ORGANIZATION, "/filter/topLevelOrganizations/id",
+            SCIENTIFIC_INDEX, "/filter/scientificIndex/year"
```

Comment:
It was really hard to find this thing. It seems like it must include aggregation path so aggregation does not filters out from original OpenSearch response?
Can this thing be made more visible?

Reply:
Agree! I am willing to try any suggestions

Reply:
What about implementing something like this when mapping sws response to PagedSearch resposne?
Instead of:

```
.withAggregations(response.getAggregationsStructured())
                .build();
```

```
.withAggregations(response.keepRequiredAggregations(REQUIRED_AGGREGATIONS).getStructuredAggregations())
                .build();
```

It makes it visible early in mapping process that we are filtering something out.

Reply:
If you agree, i can try to do something similar in next pr.